src/main/perl/Filesystem.pm

```diff
@@ -155,6 +156,83 @@ sub remove_if_needed
     return $?;
 }
 
+
+=pod
+
+=head2 get_disk_uuid
+
+Fetches the (PART)UUID of the device with blkid 
+
```

Comment:
mention it returns undef if it can't find any uuid. also mention what type is / can be (and check it)